string session_id
string peer_id
string label
string encoding
uint8[] buffer
---
